js_machine_front/src/scenes/communityEvents/components/eventsContent.tsx

```diff
@@ -6,18 +6,18 @@ interface Props {
   events: Event[];
 }
 
-export const EventsContent: React.FC<Props> = memo(({ events }) => {
+export const EventsContent: React.FC<Props> = memo(({events}) => {
   return (
     <div className="eventsContent">
-      <div className="eventsContent__wrapper">
-        {events.map((event: Event) => {
-          const eventDate: number = new Date(event.date).getDate();
-          const eventMonth: string = new Date(event.date)
-            .toLocaleString('ru', { month: 'short' })
-            .toUpperCase();
+      {events.map((event: Event) => {
+        const eventDate: number = new Date(event.date).getDate();
+        const eventMonth: string = new Date(event.date)
+          .toLocaleString('ru', {month: 'short'})
+          .toUpperCase();
 
-          return (
-            <div key={event.id} className="eventsContent__element">
+        return (
```

Comment:
in future need to have separate component for single event (performance improvement)